packages/sdk-cli/src/api/__snapshots__/baseAPI.test.ts.snap

```diff
@@ -1,3 +1,17 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`apiFetch() rejects if no auth header is available 1`] = `"Fetch of fakeAPI failed: no stored auth token"`;
+
+exports[`assertOK() rejects on status code 400 with a non-JSON body 1`] = `"Fetch of undefined returned status 400 Status 400"`;
```

Comment:
`Fetch of undefined`? Please update the test fixtures to include some mock URL so the tests validate that the error message is interpolating properly.